CleverPushLocation/Source/CleverPushLocation.m

```diff
@@ -26,6 +35,12 @@ + (void)init {
                                 CLRegion *region = [[CLCircularRegion alloc]initWithCenter:center
                                                                                     radius:[[geoFence objectForKey:@"radius"] longValue]
                                                                                 identifier:[geoFence valueForKey:@"_id"]];
+                                double num = [[geoFence objectForKey:@"delay"]doubleValue];
+                                NSMutableDictionary* dataDic = [NSMutableDictionary dictionaryWithObjectsAndKeys:
+                                                                [NSNumber numberWithDouble:num], @"delay",
```

Comment:
If `delay` is nil, the NSDictionary does not include `region` I think, as `nil` will probably mark the end of the dict.

Reply:
Done